api/release_notes/2019-release-notes.md

```diff
@@ -31,10 +57,10 @@ rules/:rule_id?include=rule_components
 
 ### 3) Rule Components belong to Properties now
 
-Rule Components used to exist as children of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
+Rule Components used to exist as a child of a Rule.  That made it impossible to reuse the same components across multiple rules.  In order to enable us to add this capability in the future, Rule Components need to move out from under a Rule and live at the Property level.
```

Comment:
Did you mean to change this back? or maybe you're just accidentally overwriting the change?

Reply:
I think `children` is grammatically correct.

Reply:
Seeing some weirdness, I'm sure it's user error.  Changed to children.